charts/base/templates/deployment.yaml

```diff
@@ -36,16 +36,14 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "base.serviceAccountName" . }}
-      securityContext:
-        {{- toYaml .Values.podSecurityContext | nindent 8 }}
       {{- if .Values.initContainers }}
       initContainers:
       {{-  toYaml .Values.initContainers | nindent 6 }}
       {{- end }}
+      securityContext:
+        {{- toYaml .Values.securityContext | nindent 12 }}
```

Comment:
it seems here we need 8 instead of 12 white-spaces,

why we had two podSecurityContext and securityContext? and why it is ok to have one named securityContext? it 
seems one is deployment/pod level and another is container level config,

if we are gonna to remove podSecurityContext, can we remove it from variables(values.yaml) list also?

what variable is Values.ports? is it some boolean? can we have it in variables list and also can we set backword compatible default value to "true" ?